ui/Screen/Profile.svelte

```diff
@@ -48,12 +44,6 @@
       href: path.profileWallet(),
       looseActiveStateMatching: false,
     },
-    {
-      icon: Icon.Settings,
-      title: "Settings",
-      href: path.profileSettings(),
```

Comment:
We should delete this route in `path.ts`.

Reply:
:+1: